docs/whats-new/stravalib-2.md

```diff
@@ -1,5 +1,14 @@
 # Stravalib V2 Migration Guide
 
+:::{toctree}
+:hidden:
+:caption: What's New
+
+What's New <self>
+Changelog <changelog>
+blah </reference/index>
```

Comment:
What about this?

Reply:
😆 @jsamoocha you're being nice. that was a mistake. the toctree wasn't populating and i was testing it by adding additional items to see if it was a changelog issue again. then i forgot to remove. good catch!